### PR TITLE
carbon-now : add page

### PR DESCRIPTION
**Pull request by charitarthchugh**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/carbon-now.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# carbon-now
+
+> Create beautiful images of code.
+> More at <https://github.com/mixn/carbon-now-cli> and <https://carbon.now.sh>.
```
Comment (schneiderl) [inline pages/common/carbon-now.md:4]: I'm unsure how we go with this. Do we have any examples of multiple `More Information` links? Maybe we should remove one of the links. I'll wait for other comments about this. 
Either way, even if we decide to go with both links, it should be as below: 
```suggestion
> More information: <https://github.com/mixn/carbon-now-cli> and <https://carbon.now.sh>.
```

File: pages/common/carbon-now.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Create image from file using default settings:
+
+`carbon-now {{file}}`
+
+- Create image from text in clipboard using default settings:
+
+`carbon-now --from-clipboard`
+
+- Create image standard input using default settings:
```
Comment (schneiderl) [inline pages/common/carbon-now.md:14]:
```suggestion
- Create an image from standard input using default settings:
```

File: pages/common/carbon-now.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# carbon-now
+
+> Create beautiful images of code.
+> More at <https://github.com/mixn/carbon-now-cli> and <https://carbon.now.sh>.
+
+- Create image from file using default settings:
+
+`carbon-now {{file}}`
+
+- Create image from text in clipboard using default settings:
```
Comment (schneiderl) [inline pages/common/carbon-now.md:10]:
```suggestion
- Create an image from a text in clipboard using default settings:
```

File: pages/common/carbon-now.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+# carbon-now
+
+> Create beautiful images of code.
+> More at <https://github.com/mixn/carbon-now-cli> and <https://carbon.now.sh>.
+
+- Create image from file using default settings:
```
Comment (schneiderl) [inline pages/common/carbon-now.md:6]:
```suggestion
- Create an image from a file using default settings:
```

File: pages/common/carbon-now.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+- Create image from text in clipboard using default settings:
+
+`carbon-now --from-clipboard`
+
+- Create image standard input using default settings:
+
+`{{input}} | carbon-now`
+
+- Create images interactively for custom settings and optionaly save a preset:
```
Comment (schneiderl) [inline pages/common/carbon-now.md:18]: What exactly does it mean to `optionally` save a preset? Does the command below saves a preset? 
I'm unsure this clearly states the functionality of the command.
```suggestion
- Create images interactively for custom settings and optionally save a preset:
```

File: pages/common/carbon-now.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+- Start at a specified line of text:
+
+`carbon-now -s {{line}} {{file}}`
+
+- End at a specific line of text:
+
+`carbon-now -e {{line}} {{file}}`
+
+- Open image in browser instead of saving:
```
Comment (schneiderl) [inline pages/common/carbon-now.md:34]:
```suggestion
- Open image in a browser instead of saving:
```

**Review by schneiderl — CHANGES_REQUESTED** [review]

File: pages/common/carbon-now.md (hunk 1 above)
Comment (navarroaxel) [reply pages/common/carbon-now.md:4]: I prefer only 1 link, the website doesn't have information about the CLI app, so I vote for this: `https://github.com/mixn/carbon-now-cli`

Comment (charitarthchugh) [reply pages/common/carbon-now.md:4]: Ok, I agree with @navarroaxel here.

Comment (charitarthchugh) [reply pages/common/carbon-now.md]: I  have removed the `https://cabon.now.sh` link

**Review by navarroaxel — APPROVED** [review]

**Comment by schneiderl** [comment]
@charitarthchugh: I'm sorry I'm getting over this again but I think I've caught a mistake on my review. 
According to the comments on #4276, there is no problem with having multiple links on the More Information tag. 
If you want to add the link again, there is no problem with it.

Sorry for the mistake.

**Review by schneiderl — APPROVED** [review]

**Pull request merged**